spec/models/manageiq/providers/foreman/provider_spec.rb

```diff
@@ -58,4 +58,16 @@
       expect(CustomizationScript.count).to   eq(0)
     end
   end
+
+  describe "#save" do
+    it "will update the name for the manager" do
+      provider = FactoryGirl.create(:provider_foreman, :zone => FactoryGirl.create(:zone), :name => 'Old Name')
+      provider.save
+      expect(provider.configuration_manager.name).to eq('Old Name Configuration Manager')
```

Comment:
It feels a little strange to me that we're doing this, but since we are, shouldn't we test that all managers are updated?